src/applications/personalization/profile/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -134,6 +134,12 @@ describe('actions/paymentInformation', () => {
           expect(recordEventSpy.secondCall.args[0].event).to.equal(
             'profile-get-direct-deposit-retrieved',
           );
+          expect(
```

Comment:
Do we want to add a test that checks for this value to be `false`?

Reply:
Sure, added!